lib/addons/p5.dom.js

```diff
@@ -2158,6 +2158,69 @@
    * @method speed
    * @param {Number} [speed]  speed multiplier for element playback
    * @return {Number|Object|p5.MediaElement} current playback speed or p5.MediaElement
+   * @example
+   * <div class="norender"><code>
+   * //this example shows playback speed
+   * //twice , half and reverse playback
+   * 
+   * //We will store the p5.MediaElement
+   * //object in here
+   * var ele;
+   * function setup() {
+   * 
+   * createCanvas(710, 400);
+   * //Here we create a p5.MediaElement object
+   * //using the createAudio() function.
+   * ele = createAudio('assets/beat.mp3');
+   * background(200);
+   *
+   * button = createButton('2x speed');
+   * button.position(100, 68);
+   * button.mousePressed(twice_speed);
+   *
+   * button = createButton('half speed');
+   * button.position(200, 68);
+   * button.mousePressed(half_speed);
+   *
+   * button = createButton('reverse play');
+   * button.position(300, 68);
+   * button.mousePressed(reverse_speed);
+   *
+   * button = createButton('STOP');
+   * button.position(400, 68);
+   * button.mousePressed(stop_song);
+   *
+   * button = createButton('PLAY!');
+   * button.position(500, 68);
+   * button.mousePressed(play_speed);
+   *
+   * }
+   * // play song at 2x speed
+   * function twice_speed() {
+   *  ele.play();
```

Comment:
could we just call ele.loop() in setup, and then just the ele.speed() lines in these functions?